File: essential/resource-extractor-services/src/main/java/ir/ac/iust/dml/kg/resource/extractor/services/web/Jackson2ObjectMapperPrettier.java
package ir.ac.iust.dml.kg.resource.extractor.services.web;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;

@Configuration
public class Jackson2ObjectMapperPrettier {
    @Bean
    public Jackson2ObjectMapperBuilder jacksonBuilder() {
        Jackson2ObjectMapperBuilder builder = new Jackson2ObjectMapperBuilder();
        builder.indentOutput(true);
        return builder;
    }
}

File: mapper/runner/src/main/java/ir/ac/iust/dml/kg/mapper/runner/web/Jackson2ObjectMapperPrettier.kt
package ir.ac.iust.dml.kg.mapper.runner.web

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder

@Configuration
open class Jackson2ObjectMapperPrettier {
  @Bean
  open fun jacksonBuilder(): Jackson2ObjectMapperBuilder {
    val builder = Jackson2ObjectMapperBuilder()
    builder.indentOutput(true)
    return builder
  }
}